### PR TITLE
Some updated raid mappings

### PR DESCRIPTION
**Pull request by liach**
Now mojang offically call the raid leaders "captains"

Signed-off-by: liach <liach@users.noreply.github.com>

### CODE REVIEW

**Comment by asiekierka** [comment]
[[ merge after #610 - i see at least two lines' worth of overlap ]]

**Comment by liach** [comment]
The final content after the diff is the same (we chose same names for the criteria triggers), so there will be no merge conflict.

File: mappings/net/minecraft/world/ChunkUpdateState.mapping [hunk 1]
```diff
@@ -6,5 +6,5 @@ CLASS cic net/minecraft/world/ChunkUpdateState
 	METHOD a getAll ()Lit/unimi/dsi/fastutil/longs/LongSet;
 	METHOD a add (J)V
 	METHOD b contains (J)Z
-	METHOD c isRemaing (J)Z
+	METHOD c isRemaining (J)Z
```
Comment (asiekierka) [inline mappings/net/minecraft/world/ChunkUpdateState.mapping:9]: Sounds a bit vague to me

Comment (liach) [reply mappings/net/minecraft/world/ChunkUpdateState.mapping:9]: just a typo fix. this should be block updates for unloaded chunks (like a task from a new vanilla command)

**Review by Gegy — APPROVED** [review]

**Pull request merged**